docs/index.md

```diff
@@ -169,8 +213,8 @@ and *authentication*.
 
 Property | Type | Descripstion
 ------------ | ------------- | -------------
-*api_version* | [**VersionApi**](VersionApi.md)  | Public APIs for querying supported REST versions. |
-*authentication* | [**AuthenticationApi**](AuthenticationApi.md)  | Login/logout APIs. |
+*api_version* | [**VersionApi**](docs/VersionApi.md)  | Public APIs for querying supported REST versions. |
+*authentication* | [**AuthenticationApi**](docs/AuthenticationApi.md)  | Login/logout APIs. |
```

Comment:
Several unintended changes to this file along these lines. The linked documents are in the same folder as this `index.md`, so having `docs/` here is incorrect.
Will fix and update commit.

Reply:
Fixed up the links here. 👍